.chainsaw.yaml

```diff
@@ -0,0 +1,13 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/kyverno/chainsaw/main/.schemas/json/configuration-chainsaw-v1alpha1.json
+apiVersion: chainsaw.kyverno.io/v1alpha1
+kind: Configuration
+metadata:
+  name: configuration
+spec:
+  skipDelete: true
+  template: true
+  timeouts:
+    assert: 6m40s
```

Comment:
[q] how did you come up with these magic numbers? are they based on the current CI time + some margin?

Reply:
to be honest I just copied them now, I will do some polishing and proper descriptions of the changes in the next day (due to lack of time)